kubernetes/spack/internal-spack-io/gh-gl-sync/cron-jobs.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: gh-gl-sync
+  namespace: spack
+spec:
+  schedule: "*/1 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+          - name: sync
+            image: alpine
+            imagePullPolicy: IfNotPresent
+            command: ["sh", "/scripts/entrypoint.sh"]
+            volumeMounts:
+              - name: scripts
+                mountPath: "/scripts"
+              - name: secrets
+                mountPath: "/secrets"
+          volumes:
+            - name: scripts
+              configMap:
+                name: gh-gl-sync
+                defaultMode: 0700
+            - name: secrets
+              secret:
+                secretName: gh-gl-sync
+                defaultMode: 0700
+          nodeSelector:
+            "beta.kubernetes.io/instance-type": "t2.medium"
```

Comment:
This has almost no resource requirements and should be fine even on a `t2.nano`

Reply:
Good point. Correct me if I'm wrong Omar, but I believe our "general" pool is t2.medium. So since we have these instances sitting around anyway, it probably doesn't hurt to run a lightweight cronjob there.